packages/react/src/components/iconBadge/IconBadge.tsx

```diff
@@ -51,16 +49,12 @@ const TypeColorMapping: Record<IconBadgeType, string> = {
     [IconBadgeType.Major]: 'mod-major',
 };
 
-export const IconBadge: FunctionComponent<IconBadgeProps> = ({
-    svgName,
-    type,
-    size = IconBadgeSize.Medium,
-    svgClass,
-    className,
-}) => (
-    <Svg
-        className={classNames('icon-badge', SizeClassMapping[size], TypeColorMapping[type], className)}
-        svgName={svgName}
-        svgClass={classNames('icon align-middle', SizeClassMapping[size], svgClass)}
-    />
-);
+export const IconBadge: FunctionComponent<IconBadgeProps> = ({icon, type, size = IconBadgeSize.Medium, className}) => {
+    const IconName = icon;
+
+    return (
+        <div className={classNames('icon-badge', SizeClassMapping[size], TypeColorMapping[type], className)}>
+            <IconName height={24} className={classNames('align-middle')} />
+        </div>
+    );
```

Comment:
I'm not sure I see the value of changing `icon` to `IconName`
```suggestion
    return (
        <div className={classNames('icon-badge', SizeClassMapping[size], TypeColorMapping[type], className)}>
            <icon height={24} className={classNames('align-middle')} />
        </div>
    );
```

Reply:
Typescript complains if the name of the component is not starting with a capital letter